include/uapi/sound/sof/abi.h

```diff
@@ -26,7 +26,7 @@
 
 /* SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 5
+#define SOF_ABI_MINOR 6
```

Comment:
@ranj063 @lgirdwood 
There is still a gap in ABI version between kernel (3.5.0) and SOF (3.6.0)from
https://github.com/thesofproject/sof/pull/1036

```suggestion
#define SOF_ABI_MINOR 7
```

Reply:
Ack MINOR 7 should be used - see ABI classification.